packages/components/addon/components/hds/form/checkbox/group.hbs

```diff
@@ -0,0 +1,9 @@
+<fieldset class={{this.classNames}}>
+  {{yield (hash Legend=(component "hds/form/legend" _contextualClass="hds-form-group__legend"))}}
+  <div class="hds-form-group__control-fields-wrapper">
+    {{yield
+      (hash Checkbox::Field=(component "hds/form/checkbox/field" _contextualClass="hds-form-group__control-field"))
+    }}
+  </div>
+  {{yield (hash Error=(component "hds/form/error" _contextualClass="hds-form-group__error"))}}
+</fieldset>
```

Comment:
For discussion: I'm thinking we may be able to abstract this wrapper (fieldset + legend) into a utility component.
```
<Hds::Form::Fieldset>
  {{yield
    (hash Checkbox::Field=(component "hds/form/checkbox/field" _contextualClass="hds-form-group__control-field"))
  }}
</Hds::Form::Fieldset>
```
The reason I'm suggesting this is because we'll likely introduce more logic here that could be shared between group components. A few examples that I can think of:
 - We may want to introduce a helper text at the group level (below the legend)
 - We may need to create relationship between the fieldset and the error message (e.g. `<fieldset aria-describedby="error-message-id">` (and potentially for the helper text).

I appreciate this suggestion can be addressed in a further iteration, but it's something for us to consider.

Reply:
https://app.asana.com/0/1202008719910219/1202406318316728